doc/user_guide/metadata_structure.rst

```diff
@@ -88,15 +97,30 @@ original_filename
     original file.
 
 time
-    type: datetime.time
+    type: Str
 
-    The acquistion or creation time.
+    The acquisition or creation time in ISO 8601 time format.
 
 date
-    type: datetime.time
+    type: Str
+
+    The acquisition or creation date in ISO 8601 date format
 
-    The acquistion or creation date.
 
+authors
+    type: Str
+
+    The authors of the data, e. g. surname1, name1 and surname2, name2, etc.
```

Comment:
Instead of e.g. I would be more strict: "in Latex format: Surname1, Name1 and Surname2, Name2 and ..."